controllers/helpers_test.go

```diff
@@ -84,3 +90,154 @@ func getClusterScope(awsCluster infrav1.AWSCluster) (*scope.ClusterScope, error)
 		},
 	)
 }
+
```

Comment:
Suggestion: In functions like `mockedCreateLBCalls()` and `mockedDeleteLBCalls()` at most places we have redundant code. We can try to reduce it. Just have an idea :-> to create a variable for functions input and output.
Eg:
```
describeLBInput:= &elb.DescribeLoadBalancersInput{
		LoadBalancerNames: aws.StringSlice([]string{"test-cluster-apiserver"}),
}
```
and then can use this input in both the `mock delete and mock create functions`. Wdyt?

Reply:
Done